pkg/profiler/cpu/bpf_metrics.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cpu
+
+import (
+	"bufio"
+	"bytes"
+	_ "embed"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+)
+
+func GetBPFMetrics(logger log.Logger, m *bpf.Module, p procfs.Proc, mapMetrics *prometheus.GaugeVec) {
```

Comment:
```suggestion
func getBPFMetrics(logger log.Logger, m *bpf.Module, p procfs.Proc, mapMetrics *prometheus.GaugeVec) {
```
No need to export, as it seems. Also, PID could be enough.

One other idea worth exploring it this method to be part of the `maps.go` and `epbpfMaps` type.